pkg/vere/main.c

```diff
@@ -1159,9 +1160,13 @@ _cw_eval(c3_i argc, c3_c* argv[])
 {
   c3_i ch_i, lid_i;
   c3_w arg_w;
+  c3_o jam_l = c3n;
+  c3_o khan_l = c3n;
```

Comment:
Doesn't conform to naming scheme: `loobean` variables should end with `_o`. Also, it's another naming convention that variable names are three-letter strings (usually consonant-vowel-consonant). I recommend `kan_o`.

Reply:
Changed